pyxdf/pyxdf.py

```diff
@@ -611,7 +614,13 @@ def _clock_sync(
                 if range_i[0] != range_i[1]:
                     start, stop = range_i[0], range_i[1] + 1
                     e = np.ones((stop - start,))
-                    X = np.column_stack([e, np.array(clock_times[start:stop]) / winsor_threshold])
+                    X = np.column_stack(
+                        [
+                            e,
```

Comment:
If you are reformatting anyway, you could remove `e` and directly use `np.ones((stop - start,))` here.

Reply:
sorry, that was my autoformatter.

Reply:
fixed it anyways